src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countOfCalls = 0;
+
+  function getSum(firstOperand) {
+    return function(secondOperand) {
+      countOfCalls++;
+
+      if (countOfCalls >= 4 && countOfCalls % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return firstOperand + secondOperand;
+      }
```

Comment:
```suggestion
      } 
      return firstOperand + secondOperand;
```
There is no need to use else. But i think, it isn't mistake)